client/components/plans/plans-compare/index.jsx

```diff
@@ -24,7 +24,8 @@ var observe = require( 'lib/mixins/data-observe' ),
 	Card = require( 'components/card' ),
 	featuresListUtils = require( 'lib/features-list/utils' ),
 	filterPlansBySiteAndProps = require( 'lib/plans' ).filterPlansBySiteAndProps,
-	shouldFetchSitePlans = require( 'lib/plans' ).shouldFetchSitePlans;
+	shouldFetchSitePlans = require( 'lib/plans' ).shouldFetchSitePlans,
+	TransactionStepTypes = require( 'lib/store-transactions/step-types' );
```

Comment:
Good idea to use constants here. Also, super minor, but I think the first word should be lower case in this variable: `transactionStepTypes`